client.js

```diff
@@ -60,14 +97,20 @@ const isValidURL = function (url) {
   return false;
 };
 
-//GOAL:If a message does not pass validation, it should be sent to the DLQ and removed from the input topic.
+const sendMsg = (message, token) => {
+  axios
+    .post(`${URL}/message`, { message }, { headers: { "X-Gotify-Key": token } })
+    .then(console.log("sendMsg", message))
+    .catch((err) => console.log("fn sendMsg", err.response));
+};
 
-const handleDLQ = function (message) {
+const deleteMsg = function (msgId, token) {
   axios
-    .post(`${URL}/message`)
-    .then((res) => {
-      console.log(message);
-      console.log(res.data);
+    .delete(`${URL}/message/${msgId}`, {
+      headers: { "X-Gotify-Key": token },
     })
-    .catch((err) => console.log(err));
+    .then(console.log("Deleting message with id:", msgId))
+    .catch((err) => {
+      console.log("deleteMsg error");
```

Comment:
We should log the entire error message:
```suggestion
      console.log("Error deleting message:", err);
```

Reply:
Changed to:       console.log("Error deleting message:", err);